src/main.js

```diff
@@ -58,12 +53,12 @@ if (__DEV__) {
     }
 
     // Setup hot module replacement
-    module.hot.accept('./routes/index', () => {
-      setTimeout(() => {
+    module.hot.accept('./routes/index', () =>
+      setImmediate(() => {
```

Comment:
tldr setImmediate uses IO queue for scheduling (as opposed to setTimeout using call stack) so it is similar to `process.nextTick` in node and thus more performant